packages/training-link/src/TrainingLink.stories.tsx

```diff
@@ -2,13 +2,16 @@ import React from 'react';
 import { Meta, Story } from '@storybook/react';
 
 import TrainingLink, { TrainingLinkProps } from './TrainingLink';
+import { ArgsTable } from '@storybook/addon-docs';
```

Comment:
This import should come before the `TrainingLink` import. We can move it to line 3.